Packs/CommonWidgets/Scripts/GetLargestInputsAndOuputsInIncidents/GetLargestInputsAndOuputsInIncidents.yml

```diff
@@ -38,3 +38,4 @@ timeout: '0'
 type: python
 dockerimage: demisto/python3:3.9.7.24076
 fromversion: 6.0.0
+toversion: 6.1.9
```

Comment:
what will happen to someone that uses this on 6.2?  are we shower we want to make it disappear? why not just return an error message? lets talk about this.